go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/giantswarm/coredns-app
+
+go 1.13
+
+require (
+	github.com/giantswarm/e2esetup v0.2.0
+	github.com/giantswarm/e2etests v0.2.1
+	github.com/giantswarm/helmclient v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	k8s.io/apimachinery v0.17.2
+)
+
+// Workaround for https://github.com/sirupsen/logrus/issues/570
+// Additional reading: https://github.com/golang/go/issues/28489
+// Solution: https://github.com/moby/moby/issues/39302
+replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.20191113042239-ea84732a7725
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20191114100352-16d7abae0d2a
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20191114105449-027877536833
+	k8s.io/apimachinery => k8s.io/apimachinery v0.16.5-beta.1
+	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20191114103151-9ca1dc586682
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.0.0-20191114110141-0a35778df828
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20191114101535-6c5935290e33
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.0.0-20191114112024-4bbba8331835
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.0.0-20191114111741-81bb9acf592d
+	k8s.io/code-generator => k8s.io/code-generator v0.16.5-beta.1
+	k8s.io/component-base => k8s.io/component-base v0.0.0-20191114102325-35a9586014f7
+	k8s.io/cri-api => k8s.io/cri-api v0.16.5-beta.1
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.0.0-20191114112310-0da609c4ca2d
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.0.0-20191114103820-f023614fb9ea
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.0.0-20191114111510-6d1ed697a64b
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.0.0-20191114110717-50a77e50d7d9
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.0.0-20191114111229-2e90afcb56c7
+	k8s.io/kubectl => k8s.io/kubectl v0.0.0-20191114113550-6123e1c827f7
```

Comment:
Why are these replaces just some commit? Why not use a tag?

Reply:
@MarcelMue This is because of some k8s packages are not using go modules before 1.16. It would disappear after we have helmclient v1.0.0. This is only the preparation step.

Reply:
Okay - makes sense :+1: